Resources/public/js/views/ez-fieldeditview.js

```diff
@@ -229,7 +216,26 @@ YUI.add('ez-fieldeditview', function (Y) {
          * @method initializer
          */
         initializer: function () {
-            Y.eZ.TemplateBasedView.registerPartial('ez_fieldinfo_tooltip', 'fieldinfo-tooltip-ez-partial');
+            /**
+             * Set if the fieldDefinition description is active or not.
+             *
+             * @property _toggleFieldDefinitionDescription
+             * @protected
+             * @type string
+             * @default 'active'
+             */
+            this._toggleFieldDefinitionDescription = 'active';
```

Comment:
If I understand correctly based on that property, we enable or disable the handling of the field description visibility. If so, this should be a boolean value and should be called something like `_handleFieldDescriptionVisibility`.